unbxdSearch.js

```diff
@@ -1078,7 +1175,11 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
             } else if (this.options.type == "browse" && this.params['categoryId'] != undefined) {
                 url += '&category-id=' + encodeURIComponent(this.params.categoryId);
             } else if (isTypeCategory(this.options.type) && this.params.categoryId !== undefined) {
-                url += "&p=" + encodeURIComponent(this.params.categoryId);
+                url += "&p=" + this.options.mappedFields.categoryField + ":\"" + encodeURIComponent(this.params.categoryId) + "\"" + '&pagetype=boolean';
```

Comment:
Since we're planning fetch `window.UnbxdAnalyticsConf["page"]`. This will have `categoryPath` or `attribute` field already.

For instance: `categoryPath:"Bedding &amp; Bath>Happy at Home"`
Reference: `UnbxdAnalyticsConf` on this page: https://www.garnethill.com/bedding-home/event4/

On all the websites with category pages powered from Unbxd, will have analytics enabled before integration, so `UnbxdAnalyticsConf` variable will be initialized on those pages.

Also adding `this.options.mappedFields.categoryField` is redundant.

Reply:
I see that window.UnbxdAnalyticsConf is missing for some customers, ex: Ecklers. 
It is not guaranteed that this object would be present at all times, also the same file is used in different projects like 'self serve preview', where this variable won't be present.
Hence standardising all field mappings in once place for consistency.

mappedFields.categoryField indicates which field to be used for category API. Is there any other field that makes this one duplicate/redundant?

Reply:
Done, Added 'p' as configurable and category key and id to come from UnbxdAnalytticsConf